development/backend/nodejs/src/itree/ds.ts

```diff
@@ -4,9 +4,12 @@
 export interface IItem {
     name: string
     uri: string
+    developer: string
 }
 
-export interface IFlatItem extends IItem {
+export interface IFlatItem {
```

Comment:
why not put the `developer` field into `IFlatItem` at the same time

Reply:
wasn't sure if we want to put extra info in there. I just update it. Thanks for your review!